src/backend/marsha/settings.py

```diff
@@ -220,7 +220,7 @@ class Development(Base):
 class Test(Base):
     """Test environment settings."""
 
-    CLOUDFRONT_SIGNED_URLS_ACTIVE = False
+    CLOUDFRONT_SIGNED_URLS_ACTIVE = values.BooleanValue(False)
```

Comment:
IMO test settings don't need to be configurable. Let's enforce False so the test environment is certain. A test is able to override a setting if it needs to.
```suggestion
    CLOUDFRONT_SIGNED_URLS_ACTIVE = False
```